config/settings/base.py

```diff
@@ -338,6 +338,18 @@
         "rest_framework.authentication.SessionAuthentication",
         "rest_framework.authentication.TokenAuthentication",
     ),
+    "DEFAULT_THROTTLE_CLASSES": [
+        "rest_framework.throttling.AnonRateThrottle",
+        "scoap3.utils.throttling.APIUserRateThrottle",
+        "scoap3.utils.throttling.AdvancedUserRateThrottle",
+        "scoap3.utils.throttling.AdminUserRateThrottle",
+    ],
+    "DEFAULT_THROTTLE_RATES": {
+        "anon": "200/day",  # Default rate for unauthorized users
```

Comment:
maybe we should update to be possible to change the values through ENV variables